includes/emicdora.inc

```diff
@@ -26,10 +27,31 @@ function collationtools_get_edited_collation($document_id) {
       'version2' => $_GET['version2'],
       'diff_kind' => $_GET['diff_kind'],
   )));
-  $contents = file_get_contents($url);
-  $spans = preg_split( '#</span>#', $contents, -1, PREG_SPLIT_DELIM_CAPTURE | PREG_SPLIT_NO_EMPTY);
-  $fixed_content = collationtools_fix_calliope($spans, $_GET['diff_kind']);
-
+  $raw_contents = file_get_contents($url);
+  $contents = str_replace($br_token, '<br />', $raw_contents);
+  $dom = new DOMDocument();
+  $dom->loadHTML($contents);
+  $bodies = $dom->getElementsByTagName('body');
+  $body = $bodies->item(0);
+  // Get rid of comment nodes.
+  foreach ($dom->childNodes as $node) {
+    if ($node->nodeName == '#comment') {
+      $node->parentNode->removeChild($node);
+    }
+  }
+  $all_nodes = $body->childNodes;
+  foreach ($all_nodes as $node) {
```

Comment:
Not needed,  The text nodes shouldn't be there at all.  This is just there to get orphaned text nodes and wrap them in spans are they were meant to be.